include/variables.h

```diff
@@ -12,6 +12,7 @@ extern ActionState gPlayerActionState;
 extern PlayerAnim gPlayerAnimation;
 extern PlayerStatus gPlayerStatus;
 extern GameStatus* gGameStatusPtr[1];
+extern s32 D_80074410;
```

Comment:
Do we not have enough info to name this symbol `gRandSeed` or something?

Reply:
I wasn't sure if it was the seed itself or something involved in the seed's computation. but yeah I think that's a good name for now at least